README.md

```diff
@@ -50,7 +50,11 @@ The latter already works with [Flask](#flask).
 
 You set it up like this:
 
+% skip: next
```

Comment:
Maybe just `% skip: start` to cut out the noise and move it down as you get things working?

Reply:
I tried that but it didn't work either.

Reply:
Did you remove the `% skip: next`? I found a bug where a `%skip: start` followed by a `% skip: next` disables the `% skip: start`. I think that nesting should raise an exception so working on that.